src/contracts/PerformanceFee.sol

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {IERC20} from "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+
+import {MultiLP} from "./MultiLP.sol";
+
+/**
+ * @title Added a performance fee to an ARM with Liquidity Providers (LP)
+ * @author Origin Protocol Inc
+ */
+abstract contract PerformanceFee is MultiLP {
+    /// @notice The maximum performance fee that can be set
+    /// 10,000 = 100% performance fee
+    uint256 public constant MAX_FEE = 10000;
```

Comment:
This is probably the wrong name for this. Maybe FEE_DIVISOR?

Reply:
I'll change to `FEE_SCALE`